src/Components/ProfilePicture.js

```diff
@@ -0,0 +1,13 @@
+const ProfilePicture = (props) => {
```

Comment:
const profilepic = ({photoUrl})
...
return photoUrl && (
 <img class....

Reply:
we want to show back up as discussed on gather